### PR TITLE
Support Unifi setups with USG and expose Unifi-provided names

### PR DESCRIPTION
**Pull request by akramer**
Hey,

I have a Unifi Security Gateway and the exporter doesn't work because it Unifi returns wired clients as "stations" with no AP MAC set, and net.ParseMAC throws an error on the empty string.

This patch makes things work correctly, and also exposes the Unifi-set alias field.

### CODE REVIEW

File: stations.go [hunk 1]
```diff
@@ -33,12 +33,15 @@ type Station struct {
 	AssociationTime time.Time
 	Channel         int
 	FirstSeen       time.Time
+        // Hostname is the device-provided name
```
Comment (mdlayher) [inline stations.go]: Spacing looks a little off here.  Could just be GitHub though.

**Comment by mdlayher** [comment]
Nice.  I definitely want this package to also support the USG and UniFi switch lines, but I don't own either of those devices so I can't test myself.

I think it would also be nice to add a "connection type" identifier, so we can more clearly identify wired vs. wireless stations.

Something like:

```go
type ConnectionType int

const (
    ConnectionTypeUnknown ConnectionType = iota
    ConnectionTypeWired
    ConnectionTypeWireless
)

type Station struct {
    // other fields
    ConnectionType ConnectionType
}
```

**Comment by akramer** [comment]
The field is a bool - I added a bool field to the Stations struct. If you really want an enum I'll add one, but I think this is cleaner.

**Comment by mdlayher** [comment]
Aha, that'll do it.

Last comment: mind adding a test with `IsWired` set to true in the JSON?  Then LGTM.

**Comment by akramer** [comment]
I modified the existing test - if you wanted a separate test just for IsWired I'll add one, but I trust that Go is good at setting default values, I tested for the "code actually does something" case. :)

**Comment by mdlayher** [comment]
Yep, you did exactly what I wanted to see.  Thanks for the PR.  Will check out your exporter PR now.

**Pull request merged**